pkg/shell/indexes.js

```diff
@@ -275,14 +286,36 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
             return listItem;
         }
 
-        var menu = compiled.ordered("menu").map(links);
+        var term = $("#filter-menus").val()
```

Comment:
Can we please avoid introducing new jQuery, and use document.getElementById() and friends?